### PR TITLE
[BFD 819] - Sql script to delete positive claim ids from synthetic claims and delete positive event ids from synthetic event ids to clean up duplicate synthetic data

### PR DESCRIPTION
**Pull request by cbrunefearless**
### Change Details
This contains a script to delete positive claim ids in Inpatient and Carrier claims and postive event ids in our part d events.

### Acceptance Validation
SQL Script will only delete postive claims and positive events ids for syn data.

### Feedback Requested
Make sure the sql file is concise and does its job in order to delete postivie ids.

### External References

- [BFD-819](https://jira.cms.gov/browse/BFD-819)

### Security Implications

N/A
- [ ] new software dependencies

N/A
- [ ] altered security controls

N/A
- [ ] new data stored or transmitted

N/A
- [ ] security checklist is completed for this change

N/A
- [ ] requires more information or team discussion to evaluate security implications

### CODE REVIEW

**Review by mackec1 — DISMISSED** [review]

File: apps/bfd-model/bfd-model-rif/src/main/resources/db/scripts/Delete_Synthetic_Negative_Claims_With_PositiveClaimIds.sql [hunk 1]
```diff
@@ -0,0 +1,34 @@
+DELETE
```
Comment (jzulim) [inline apps/bfd-model/bfd-model-rif/src/main/resources/db/scripts/Delete_Synthetic_Negative_Claims_With_PositiveClaimIds.sql:1]: Can we comment and reference the ticket ID

Comment (mackec1) [reply apps/bfd-model/bfd-model-rif/src/main/resources/db/scripts/Delete_Synthetic_Negative_Claims_With_PositiveClaimIds.sql:1]: sql fine as is; presonally I can't wait to change the BeneficiryID to numeric; LIKE and/or NOT LIKE are pain points for performance!

**Review by mackec1 — APPROVED** [review]
LGTM

**Review by jzulim — DISMISSED** [review]
LGTM.

**Review by jzulim — DISMISSED** [review]
LGTM.

**Review by mackec1 — APPROVED** [review]
LGTM

**Review by mackec1 — APPROVED** [review]
LGTM

**Comment by jzulim** [comment]
Nice work

**Review by jzulim — APPROVED** [review]
LGTM

**Pull request merged**